nx.json

```diff
@@ -149,8 +149,11 @@
     "pmp-web-shared-core": {
       "tags": ["platform:web", "scope:shared", "type:core"]
     },
-    "pmp-web-auth-auth-feature": {
-      "tags": ["platform:web", "scope:auth", "type:feature"]
+    "pmp-web-auth-public": {
+      "tags": ["platform:web", "scope:auth", "name:auth-public", "type:public"]
+    },
+    "pmp-web-auth-feature": {
+      "tags": ["platform:web", "scope:auth", "name:auth-feature", "type:feature"]
```

Comment:
name is not necessary here. It is necessary only for public library to restrict that other scopes can import only from that specific one.